apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -0,0 +1,118 @@
+import React, {useState, useEffect} from 'react';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+
+export default function CurriculumQuickAssign() {
+  const [courseOfferings, setCourseOfferings] = useState(null);
+  const [decideLater, setDecideLater] = useState(false);
+  const [marketingAudience, setMarketingAudience] = useState(null);
+
+  // Retrieve course offerings on mount and convert to JSON
+  useEffect(() => {
+    fetch('/course_offerings/quick_assign_course_offerings')
+      .then(response => response.json())
+      .then(data => setCourseOfferings(data));
+  }, []);
+
+  /*
+  When toggling 'decide later', clear out marketing audience or assign one to make
+  the table appear again automatically.
+  */
+  const toggleDecideLater = () => {
+    setDecideLater(!decideLater);
+    if (marketingAudience !== '') {
+      setMarketingAudience('');
+    } else {
+      setMarketingAudience('elementary');
+    }
+  };
+
+  // When selecting a marketing audience, ensure 'decide later' is unchecked
+  const updateMarketingAudience = marketingAudience => {
+    setMarketingAudience(marketingAudience);
+    setDecideLater(false);
+  };
+
+  return (
+    <div>
+      <h2>{i18n.assignACurriculum()}</h2>
```

Comment:
nit: should this use `h3`?

Reply:
Updated! Good point- it should be a level smaller than the new section header